src/test/java/systemtests/AddCommandSystemTest.java

```diff
@@ -136,22 +135,6 @@ public void add() {
         command = PersonUtil.getAddCommand(HOON) + " " + PREFIX_TAG.getPrefix() + "friends";
         assertCommandFailure(command, AddCommand.MESSAGE_DUPLICATE_PERSON);
 
-        /* Case: missing name -> rejected */
```

Comment:
Can leave missing name in I think, names are supposed to be compulsory

Reply:
Yes thanks!